src/app.jsx

```diff
@@ -452,20 +453,30 @@ class Application extends React.Component {
                 });
         });
 
-        this.setState(prevState => ({ users: prevState.users.filter(u => u.uid !== con.uid) }));
+        // keep dummy (null) connections from other users, only remove valid ones
```

Comment:
I assume we keep `null` connections as we make connections when required?

Reply:
Yes, otherwise if we'd drop them, the users would disappear from the owner selector. This is only meant to clean up users (i.e. connections) whose podman.service gets AWOL, not for our deliberate disconnects.